extensions/blocks/google-calendar/edit.js

```diff
@@ -0,0 +1,187 @@
+/**
+ * External dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import {
+	Placeholder,
+	SandBox,
+	Button,
+	Notice,
+	ExternalLink,
+	PanelBody,
+} from '@wordpress/components';
+import { BlockIcon, InspectorControls } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import { icon } from '.';
+import { isMobile } from '../../../_inc/client/lib/viewport';
+import { extractAttributesFromIframe, IFRAME_REGEX, URL_REGEX } from './utils';
+
+class GoogleCalendarEdit extends Component {
```

Comment:
We have been using functional components for our newer jetpack blocks, which aligns with how Gutenberg is doing it. Not worth changing it now but worth being aware of for the future.